ably/realtime_conn.go

```diff
@@ -496,21 +494,32 @@ func (c *Connection) ErrorReason() *ErrorInfo {
 	return c.errorReason
 }
 
+// Deprecated: this property is deprecated, use CreateRecoveryKey method instead.
```

Comment:
This should be logged as well so that anyone using it is notified, it's unlikely users will see this comment.

Reply:
Then again, if this is now just calling `CreateRecoveryKey`, perhaps just don't rename the method, then nothing needs to be deprecated?

Reply:
Actually this needs to be deprecated as per spec

Reply:
Ok, so should we log the deprecation warning? Otherwise users will likely never know this is deprecated.

Reply:
Ohkay, I thought compiler does give deprecation warning 🤔